virtme_ng/version.py

```diff
@@ -13,7 +14,7 @@ def get_version_string():
         # Get the version from git describe
         version = (
             check_output(
-                "git describe --always --long --dirty",
+                "cd %s && git describe --always --long --dirty" % os.path.dirname(__file__),
```

Comment:
Good fix! 
Should you eventually check that the parent directory has a `.git` directory or file not to look at the wrong repo?